roles/ceph-mon/tasks/deploy_monitors.yml

```diff
@@ -51,7 +51,7 @@
     client_admin_ceph_authtool_cap:
```

Comment:
I wonder why we should use this set_fact, I think it could be moved somewhere in ceph-defaults.
at least, if we really want to keep it, I think it should be moved into ceph-facts